src/components/organisms/radio-group/index.tsx

```diff
@@ -100,11 +107,14 @@ const SimpleItem: React.FC<RadioGroupSimpleItemProps> = ({
           )}
         />
       </RadioGroupPrimitive.Item>
-      {label && (
+      <div className="ml-small inter-base-regular cursor-pointer w-full">
```

Comment:
Nit: Can we add the `indicator` class to the `RadioGroupPrimitive.Indicator` element. 

This will just add a violet-60 outline to the radio button as well as the dot in the center.

I cant place this comment on the right line but it's on line 105